x-pack/plugins/threat_intelligence/public/modules/cases/components/add_to_new_case/add_to_new_case.tsx

```diff
@@ -57,9 +59,11 @@ export const AddToNewCase: VFC<AddToNewCaseProps> = ({
     attachmentMetadata
   );
 
-  // disable the item if there isn't an indicator name
+  // disable the item if there is no indicator name or if the user doesn't have the right permission
   // in the case's attachment, the indicator name is the link to open the flyout
-  const disabled: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
+  const invalidIndicatorName: boolean = attachmentMetadata.indicatorName === EMPTY_VALUE;
```

Comment:
Can this logic be shared between adding to existing case and creating a new one?

Reply:
I thought about it but ended up not doing it I don't remember why... I'll make the change!